models/sprint.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, Sequelize) => {
+  class Sprint extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+       this.hasMany(models.Task, {
+        foreignKey: 'sprintId',
+        targetKey: 'id'
+      });
+       this.belongsTo(models.Tag, {
+        foreignKey: 'tagId',
+        targetKey: 'id'
+      });
+    }
+  }
+  Sprint.init({
+    name: {
```

Comment:
check alignment of your code.
use `prettier` to format your code before commiting